api/spec/controllers/mno_enterprise/jpi/v1/admin/organizations_controller_spec.rb

```diff
@@ -157,7 +157,7 @@ def partial_hash_for_arrears(arrear)
         # Track the api call
         @api_call = false
         stub = -> { @api_call = true; from_api(org_invite) }
-        api_stub_for(post: "/organizations/#{organization.id}/org_invites", response: stub)
+        api_stub_for(post: "/org_invites", response: stub)
```

Comment:
We need a spec for when the invitation is not saved.
I think `persisted?` is just checking the `id` so you can have a stub without id for this case.